grails-app/views/plugin/Geneprint.gsp

```diff
@@ -19,7 +19,7 @@
                 <span>Select a High Dimensional Data node from the Data Set Explorer Tree and drag it into the box.</span>
                 <div id='divIndependentVariable' class="queryGroupIncludeSmall highDimBox"></div>
                 <div class="highDimBtns">
-                    <button type="button" onclick="highDimensionalData.gather_high_dimensional_data('divIndependentVariable')">High Dimensional Data</button>
+                    <button type="button" onclick="highDimensionalData.gather_high_dimensional_data('divIndependentVariable', false, false)">High Dimensional Data</button>
```

Comment:
What checks are still active for geneprint? Do you need any HD node validation at first place?